src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -242,6 +242,52 @@ type SynAccess =
 
     member Range: range
 
+/// Represents additional information for `get, set` syntax
+[<NoEquality; NoComparison; RequireQualifiedAccess>]
+type GetSetKeywords =
+    | Get of range
+    | Set of range
+    | GetSet of get: range * getterAccess: SynAccess option * set: range * setterAccess: SynAccess option
+
+    member Range: range
+
+/// Represents additional information for SynMemberDefn.AutoProperty
+[<NoEquality; NoComparison>]
+type SynMemberDefnAutoPropertyTrivia =
```

Comment:
Why did you move this? This feels very off.

Reply:
Because I moved `GetSetKeywords` to be after of `SynAccess`

Reply:
I do not agree here. The additional SynAccess is not trivia.

Reply:
The additional `SynAccess` was firstly saved into `GetSetKeywords` by parser, then be passed to `mkAutoPropDefn` , so `GetSetKeywords` and types related to it need to be moved after `SynAccess`.

Or could you provide a way to avoid this?

Reply:
You can add it as an extra argument to `mkAutoPropDefn`. This means you'll have to move some things around in the parser, but the goal is to leave the trivia alone because what you're adding isn't considered trivia. Think of trivia as stuff that's nice for tools to use but not something the compiler needs to have to type-check.

Reply:
will rework this

Reply:
completed

Reply:
Thank you 💪